plugin/depsusage/packagehint.go

```diff
@@ -30,3 +33,67 @@ func resolvePythonPackageHint(s string) string {
 	}
 	return s
 }
+
+func resolveGoPackageHint(moduleName string) string {
+	moduleName = strings.Trim(moduleName, "/")
+	if moduleName == "" {
+		return ""
+	}
+
+	parts := strings.Split(moduleName, "/")
+	if len(parts) == 0 {
+		return ""
+	}
+
+	domain := strings.Trim(parts[0], " ")
+
+	domainWiseQualifierCount := map[string]int{
+		"github.com":    3,
+		"bitbucket.org": 3,
+		"gopkg.in":      3,
+		"gocv.io":       3,
+		"golang.org":    3,
+		"go.etcd.io":    3,
+	}
+
+	versionSuffixRegexp := regexp.MustCompile(`v\d+$`)
```

Comment:
`MustCompile` will panic in case of error. Lets take this in `init() { ... }` so that it fails fast at program initiatlization only in case of issue instead of at runtime